### PR TITLE
updates from build run through:

### PR DESCRIPTION
**Pull request by chatch**
- seems the path to parity has changed to /parity/parity although the Dockerfile on dockerhub does not reflect this
- example dapp updated to latest truffle 3 as it failed to compile

### CODE REVIEW

File: dapps/example/truffle.js [hunk 1]
```diff
@@ -1,22 +1,9 @@
...
-      "javascripts/app.js"
-    ],
-    "app.css": [
-      "stylesheets/app.css"
-    ],
-    "images/": "images/"
-  },
   networks: {
-      "ropsten": {
-          host: "parity-ropsten",
```
Comment (gnidan) [inline dapps/example/truffle.js:14]: Mind leaving the ropsten network config here? To show how truffle apps can make use of that parity container

**Review by gnidan — CHANGES_REQUESTED** [review]
Looks good apart from the one change. I will try out the changes when I get home from work later

File: dapps/example/truffle.js (hunk 1 above)
Comment (chatch) [reply dapps/example/truffle.js:14]: sure. I must of have blown that away when i copied in all the latest "truffle init" app source.

**Comment by chatch** [comment]
ropsten in truffle.js restored

**Review by gnidan — APPROVED** [review]

**Pull request merged**